### PR TITLE
extension routes and layouts

### PR DESCRIPTION
**Pull request by bakhrom-akbarov**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/layouts/extension/BottomBarLayout.tsx [hunk 1]
```diff
@@ -39,4 +39,4 @@ const BottomBarLayout = ({ footer, children }: Props): JSX.Element => {
   )
 }
 
-export default BottomBarLayout
+export default BottomBarLayout
```
Comment (TarasPotsikaylo) [inline packages/blockchain-wallet-v4-frontend/src/layouts/extension/BottomBarLayout.tsx:42]: extra space

File: packages/blockchain-wallet-v4-frontend/src/routes/extensionRoutes/HomeRoutes.tsx [hunk 2]
```diff
@@ -94,4 +89,4 @@ const mapDispatchToProps = (dispatch) => ({
   router: bindActionCreators(actions.router, dispatch)
 })
 
-export default connect(null, mapDispatchToProps)(HomeRoutes)
+export default connect(null, mapDispatchToProps)(HomeRoutes)
```
Comment (TarasPotsikaylo) [inline packages/blockchain-wallet-v4-frontend/src/routes/extensionRoutes/HomeRoutes.tsx:92]: same

**Review by NikolaiYurchenko — CHANGES_REQUESTED** [review]
seems like incorrect changes in PR. Did you forget to add changes?

**Comment by bakhrom-akbarov** [comment]
closed

**Pull request closed without merging**